addons/hr_expense/models/hr_expense.py

```diff
@@ -61,6 +66,7 @@ def _compute_amount(self):
         for expense in self:
             expense.untaxed_amount = expense.unit_amount * expense.quantity
             taxes = expense.tax_ids.compute_all(expense.unit_amount, expense.currency_id, expense.quantity, expense.product_id, expense.employee_id.user_id.partner_id)
+            expense.total_amount_company_currency = expense.currency_id.compute(taxes.get('total_included'), expense.company_currency_id)
```

Comment:
Should'tn you make a expense.tax_ids.compute_all(blah) with the company currency in order to avoid rouding issue on an already-rounded value ?